Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/UserDAO.java

```diff
@@ -98,22 +101,22 @@ public int addAccountRequest(String userName) throws QuadrigaStorageException {
      *             in case there is a DB error.
      */
     @Override
-    public boolean addNewUserAccountRequest(String username, String password, String fullname, String email)
+    public boolean addNewUserAccountRequest(String username, String password, String fullname, String email, String provider, String userIdOfProvider)
             throws QuadrigaStorageException {
```

Comment:
Have not encountered database specific exception. Since the code involves database operations =>    sessionFactory.getCurrentSession().save(userRequestDTO);
sessionFactory.getCurrentSession().flush();
the methods throws QuadrigaStorageException

Reply:
```
sessionFactory.getCurrentSession().save(userRequestDTO);
sessionFactory.getCurrentSession().flush();
``` 
would throw hibernate exceptions, wouldn't it? where is the quadriga exception thrown?